### PR TITLE
[TASK] Add language selection and language util

### PR DESCRIPTION
**Pull request by xf-**
## Description

Select/change your language in meshviewer and store in localstorage.

## Motivation and Context
#61 

## How Has This Been Tested?
Working, but test again ;)

## Checklist:

- [x] My code follows the code style of this project. (CI will test it anyway and also needs approval)

### CODE REVIEW

File: scss/night.scss [hunk 1]
```diff
@@ -35,6 +35,14 @@ html {
     }
   }
 
+  .language-switch {
+    color: $color-black;
+
+    option {
+      background: $color-white;
```
Comment (genofire) [inline scss/night.scss:42]: needed?
it is already set in `_legend.js`

File: lib/utils/language.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+          LLL: 'D. MMMM YYYY HH:mm',
+          LLLL: 'dddd, D. MMMM YYYY HH:mm'
+        },
+        calendar: json.momentjs.calendar,
+        relativeTime: json.momentjs.relativeTime
+      });
+    }
+
+    window._ = new Polyglot({ locale: getLocale(), allowMissing: true });
+    helper.getJSON('locale/' + _.locale() + '.json?' + config.cacheBreaker).then(setTranslation);
```
Comment (genofire) [inline lib/utils/language.js:53]: should the `getJSON` not be inside of the reloadable function?

Comment (genofire) [reply lib/utils/language.js:53]: A okay, `location.reload()` is the solution.

**Review by genofire — APPROVED** [review]

File: scss/night.scss (hunk 1 above)
Comment (genofire) [reply scss/night.scss:42]: okay the colors are redefined here

**Review by genofire — APPROVED** [review]

File: scss/night.scss (hunk 1 above)
Comment (xf-) [reply scss/night.scss:42]: option doesn't inherit its color. Systemcolor will be used

File: lib/utils/language.js (hunk 2 above)
Comment (xf-) [reply lib/utils/language.js:53]: the plugin for translations has no index over all translations. It is set in local-storage This loads a translation and sets locale if a instance of Language is started. Second load is to ensure it is in the cache :)

Comment (genofire) [reply lib/utils/language.js:53]: i thought so - not perfect - in my eyes, but the most language libery works this way. or load all languages.

Comment (xf-) [reply lib/utils/language.js:53]: define perfect - use ram to have everything dynamically and no reload - including virtual-dom stuff (already really heavy on large instances). You maybe need to repaint stuff. I think a reload in JS app is OK. It is no common event. You have once your language and stay with it. (except developers)

**Pull request merged**